<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FazerTransferencia</class>
 <widget class="QDialog" name="FazerTransferencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelConta">
           <property name="text">
            <string>Conta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelQtd">
           <property name="text">
            <string>Quantidade</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="DestinoText"/>
         </item>
         <item>
          <widget class="QLineEdit" name="QtdValor"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Sair">
         <property name="text">
          <string>Sair</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConfirmaButton">
         <property name="text">
          <string>Confirmar TransferÃªncia</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
